tasks.py

```diff
@@ -21,7 +21,7 @@
     'jpg', 'gif'
 )
 STATIC_ASSETS_DIRS = (
-    'dataset', 'organization', 'post', 'reuse', 'site', 'topic', 'user'
+    'dataset', 'organization', 'post', 'reuse', 'site', 'topic', 'user', 'chunks',
```

Comment:
Nop, this is why I precised it in the description. But I can split it into another PR.